comet/archiver.py

```diff
@@ -44,129 +64,157 @@ def __init__(self, data_dump_path, scrape_interval, broker_host, broker_port):
             )
             exit(1)
 
-        self.loop = asyncio.get_event_loop()
-        self.task = self.loop.create_task(self._scrape())
```

Comment:
Why did these get removed?

Reply:
The archiver got completely rewritten. The previous one read from file what it has to archive (and used asyncio). This one (doesn't and) reads its input from lists on redis.